doc/lua_api.txt

```diff
@@ -4879,6 +4879,20 @@ Utilities
   or checking if a mod is enabled.
 * `minetest.get_modnames()`: returns a list of enabled mods, sorted alphabetically.
     * Does not include disabled mods, even if they are installed.
+* `minetest.get_game_info()`: returns a table containing information about the
+  current game. Note that other meta information (e.g. version/release number)
+  can be manually read from `game.conf` in the game's root directory.
+
+      {
+          id = string,
+          title = string,
+          author = string,
+          -- The root directory of the game
+          path = string,
+          -- The directory containing game mods
+          mods_path = string,
+      }
```

Comment:
Is this ever different from path + /mods/?

Reply:
I don't know, but it's a separate field in `SubgameSpec` that isn't available from `game.conf`.

Reply:
I removed the field from the table.